scarb/src/compiler/plugin/builtin.rs

```diff
@@ -1,25 +1,19 @@
 use std::sync::Arc;
 
 use anyhow::Result;
-use cairo_lang_defs::plugin::{InlineMacroExprPlugin, MacroPlugin};
-use cairo_lang_starknet::inline_macros::get_dep_component::{
-    GetDepComponentMacro, GetDepComponentMutMacro,
-};
-use cairo_lang_starknet::inline_macros::selector::SelectorMacro;
-use cairo_lang_starknet::plugin::StarkNetPlugin;
-use cairo_lang_test_plugin::TestPlugin;
+use cairo_lang_defs::plugin::{MacroPlugin, PluginSuite};
+use cairo_lang_starknet::starknet_plugin_suite;
+use cairo_lang_test_plugin::test_plugin_suite;
 
 use crate::compiler::plugin::{CairoPlugin, CairoPluginInstance};
 use crate::core::{PackageId, PackageName, SourceId};
 use crate::internal::to_version::ToVersion;
 
 impl CairoPluginInstance for Arc<dyn MacroPlugin> {
```

Comment:
Are we using this impl anywhere anymore? Maybe it'll be worth dropping it

Reply:
I don't think so!